### ISSUE TITLE
Tags cause Background to fail

### ISSUE THREAD

**Issue by tedlandis** [pytest-dev/pytest-bdd#160]
```bash
venv/lib/python2.7/site-packages/pytest_bdd/feature.py:290: in __init__
    "Background section can only contain Given steps", line_number, clean_line, filename)
E   FeatureError: Background section can only contain Given steps.
E   Line number: 17.
E   Line: @wip.
```
```gherkin
Background:
  Given I am on the new sale page as an associate


@wip
Scenario: Create new item by adding it to a sale
```

As you can see above, the Background only contains Given steps but the tag (@wip) on the first Scenario is being incorrectly included in the Background and causing an error.

As a workaround I tried to turn off strict checking using the following code but I couldn't get that to work either:
```python
@fixture(scope='session')
def pytestbdd_strict_gherkin():
    return False
```

Thanks

**Comment by The-Compiler** [comment]
I can confirm this, just ran into the same.

**Comment by The-Compiler** [comment]
PR which fixes this: #172 - @tedlandis, can you confirm the fix?
